### PR TITLE
add new pd_workshop_surveys table

### PR DESCRIPTION
**Pull request by Hamms**
for incoming new Pd::WorkshopSurveys form

### CODE REVIEW

File: dashboard/db/schema.rb [hunk 1]
```diff
@@ -505,6 +505,14 @@
     t.index ["user_id"], name: "index_pd_workshop_material_orders_on_user_id", unique: true, using: :btree
   end
 
+  create_table "pd_workshop_surveys", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+    t.integer  "pd_enrollment_id",               null: false
+    t.text     "form_data",        limit: 65535, null: false
+    t.datetime "created_at",                     null: false
+    t.datetime "updated_at",                     null: false
+    t.index ["pd_enrollment_id"], name: "index_pd_workshop_surveys_on_pd_enrollment_id", using: :btree
```
Comment (aoby) [inline dashboard/db/schema.rb]: not unique  :(

File: dashboard/db/migrate/20170509020232_create_pd_workshop_survey.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreatePdWorkshopSurvey < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_workshop_surveys do |t|
+      t.references :pd_enrollment, null: false, index: true, unique: true
```
Comment (aoby) [inline dashboard/db/migrate/20170509020232_create_pd_workshop_survey.rb]: This misleadingly doesn't actually create a unique index. See below.

In my experience, I've had to add the index separately if I want it to be unique. Here's an [example](https://github.com/code-dot-org/code-dot-org/blob/0b5917fffa6b0c9f8521517ffe8ce5a04f0ca6b3/dashboard/db/migrate/20170407201709_create_pd_workshop_material_orders.rb#L27-L27)

**Comment by ashercodeorg** [comment]
Presumably the timestamp on schema.rb should also be moved forward (matching the migration filename timestamp) as part of this PR?

**Comment by Hamms** [comment]
The timestamp on schema.rb is already ahead of this migration, due to the fact that I opened it up a while before I was ready to commit it

File: dashboard/db/migrate/20170509020232_create_pd_workshop_survey.rb (hunk 2 above)
Comment (Hamms) [reply dashboard/db/migrate/20170509020232_create_pd_workshop_survey.rb]: Upon further investigation, it looks like all you actually have to do is is pass `{unique: true}` as the argument to `index`! TIL

Comment (aoby) [reply dashboard/db/migrate/20170509020232_create_pd_workshop_survey.rb]: Nice. TIL too :)

**Pull request merged**